docs/template-nodepool-cr.md

```diff
@@ -13,16 +13,16 @@ To create the manifests for a new node pool, use this command:
 
 Here are the supported flags:
 
-  - `--availability-zones` - list of availability zones to use, instead of setting a number. Use comma to separate values.
+  - `--availability-zones` - list of availability zones to use, instead of setting a number. Use comma to separate values. (e.g. `eu-central-1a,eu-central-1b`)
```

Comment:
Should we not call it failure domain  to be closer to the cluster api?

Reply:
well we align with k8s terminology but IMO we move away for end-user, we have been naming this `availability zones` since the beginning. If we do a consensual change cross-company we can change terminology in all places (UI, ctls, docs) as we did with BYOC. But I will not start here